client/reader/following-manage/feed-search-results.jsx

```diff
@@ -22,14 +23,24 @@ const FollowingManageSearchFeedsResults = ( {
 	fetchNextPage,
 	forceRefresh,
 	searchResultsCount,
+	query,
 } ) => {
+	const isError = !! ( query && searchResults.length === 0 );
+	const classNames = classnames( 'following-manage__search-results', {
+		'is-error': !! isError,
```

Comment:
If this is intended for `no-results` only, maybe class name should be `is-empty` instead?